bokehjs/src/coffee/embed.ts

```diff
@@ -33,6 +33,7 @@ declare interface CommMessage {
 declare interface Comm {
   target_name: string
   on_msg: (msg: CommMessage) => void
+  onMsg: Function
```

Comment:
should be ``(this: Document, receiver: Receiver, comm_msg: CommMessage) => void`` instead of Function. Or (better yet), define an interface and share it here and with ``_handle_notebook_comms``.